scripts/kind-e2e/e2e.sh

```diff
@@ -87,10 +87,46 @@ function kind_import_images() {
         kind --name cluster${i} load docker-image submariner:local
         kind --name cluster${i} load docker-image submariner-route-agent:local
     done
+
+    if [ ! -z "${global_CIDRs[cluster1]}" ]
```

Comment:
Why not put this conditional inside the loop on L85-L89? (Obviously check `"${global_CIDRs[cluster${i}]}"` in that case)

As a matter of fact you can check "$globalnet" = "true" which would be cleaner IMHO

Reply:
Since I have to tag the image along with loading them into clusters, I used a separate if check.
I can update it, not an issue.